Namespace(batch_size=4, beta1=0.0, beta2=0.99, checkpoint_dir='expr/checkpoints', ds_iter=100000, eval_dir='expr/eval', eval_every=50000, f_lr=1e-06, hidden_dim=512, img_size=256, inp_dir='assets/representative/custom/female', lambda_cyc=1.0, lambda_ds=2.0, lambda_reg=1.0, lambda_sty=1.0, latent_dim=16, lm_path='expr/checkpoints/celeba_lm_mean.npz', lr=0.0001, mode='train', num_domains=10, num_outs_per_domain=10, num_workers=8, out_dir='assets/representative/celeba_hq/src/female', print_every=1000, randcrop_prob=0.5, ref_dir='assets/representative/celeba_hq/ref', result_dir='expr/results', resume_iter=50000, sample_dir='expr/samples', sample_every=5000, save_every=5000, seed=777, src_dir='assets/representative/celeba_hq/src', style_dim=64, total_iters=200000, train_img_dir='../GANdata/train', val_batch_size=8, val_img_dir='../GANdata/train', w_hpf=0.0, weight_decay=0.0001, wing_path='expr/checkpoints/wing.ckpt')
Number of parameters of generator: 33892995
Number of parameters of mapping_network: 9004672
Number of parameters of style_encoder: 21179584
Number of parameters of discriminator: 20856394
Initializing generator...
Initializing mapping_network...
Initializing style_encoder...
Initializing discriminator...
Preparing DataLoader to fetch source images during the training phase...
Traceback (most recent call last):
  File "main.py", line 182, in <module>
    main(args)
  File "main.py", line 47, in main
    num_workers=args.num_workers),
  File "/host/space0/yamamoto-k/jupyter/notebook/B4-5/stargan-v2/core/data_loader.py", line 108, in get_train_loader
    dataset = ImageFolder(root, transform)
  File "/usr/local/anaconda3/lib/python3.7/site-packages/torchvision/datasets/folder.py", line 206, in __init__
    is_valid_file=is_valid_file)
  File "/usr/local/anaconda3/lib/python3.7/site-packages/torchvision/datasets/folder.py", line 95, in __init__
    samples = make_dataset(self.root, class_to_idx, extensions, is_valid_file)
  File "/usr/local/anaconda3/lib/python3.7/site-packages/torchvision/datasets/folder.py", line 49, in make_dataset
    for root, _, fnames in sorted(os.walk(target_dir, followlinks=True)):
  File "/usr/local/anaconda3/lib/python3.7/os.py", line 369, in walk
    is_dir = entry.is_dir()
KeyboardInterrupt
